web/crux-ui/src/pages/products/[productId]/versions/[versionId]/deployments/[deploymentId].tsx

```diff
@@ -202,7 +226,14 @@ const DeploymentDetailsPage = (props: DeploymentDetailsPageProps) => {
         </>
```

Comment:
I know that's not your code, but I just found a bug here, you can delete secrets after a successful deployment (delete throws an exception so it's probably just a UI bug) can we talk about this?

Reply:
Fixed :)

Reply:
noice, thx